tests/unit-tests/class-newspack-ads-tests-model.php

```diff
@@ -1,112 +1,14 @@
 <?php
 /**
- * Tests the plugin management functionality.
+ * Tests the ads model functionality.
  *
  * @package Newspack\Tests
  */
 
 /**
- * Test plugin management functionality.
+ * Test ads model functionality.
  */
 class Newspack_Ads_Test_Plugin_Manager extends WP_UnitTestCase {
```

Comment:
I get why we no longer need most of these tests. Wonder if there are any tests we can add to cover the GAM interactions? e.g. checking the connection status, serializing the ad unit data (with mock data), syncing to the options table, etc.

Reply:
Added tests in 63a58779b30435b9c877b78efbb7b6d4f02a69d6 👌